experiment/dash/dash.go

```diff
@@ -3,42 +3,132 @@ package dash
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"time"
 
+	"github.com/montanaflynn/stats"
+	"github.com/neubot/dash/client"
+	neubotModel "github.com/neubot/dash/model"
 	"github.com/ooni/probe-engine/experiment"
 	"github.com/ooni/probe-engine/experiment/handler"
-	"github.com/ooni/probe-engine/experiment/mkevent"
-	"github.com/ooni/probe-engine/measurementkit"
 	"github.com/ooni/probe-engine/model"
 	"github.com/ooni/probe-engine/session"
 )
 
 const (
-	testName    = "dash"
-	testVersion = "0.7.0"
+	testName       = "dash"
+	testVersion    = "0.8.0"
+	defaultTimeout = 55 * time.Second
+	maxRuntime     = 15.0
 )
 
 // Config contains the experiment config.
-type Config struct {
-	// LogLevel is the MK log level. Empty implies "WARNING".
-	LogLevel string
+type Config struct{}
+
+type Simple struct {
+	Connect_latency   float64 `json:"connect_latency"`
+	Median_bitrate    int64   `json:"median_bitrate"`
+	Min_playout_delay float64 `json:"min_playout_delay"`
+}
+
+// TestKeys contains the test keys
+type TestKeys struct {
+	Simple `json:"simple"`
+
+	Client_resolver string `json:"client_resolver"`
+
+	// Failure is the failure string
+	Failure string `json:"failure"`
+
+	Receiver_data []neubotModel.ClientResults `json:"receiver_data"`
```

Comment:
Please, run `go vet`, install `go lint` and run `go lint`, on this specific file.

Reply:
Nothing! Do you think something is wrong here?

Reply:
No, it was just a reminder. (If you use VSCode with the Go plugin, every time you save a file, it is reformatted, linted, and vetted.)